modules/react/icon/lib/Graphic.tsx

```diff
@@ -79,23 +84,56 @@ export const graphicStencil = createStencil({
   },
 });
 
+export const graphicImageStencil = createStencil({
+  vars: {
+    width: '',
+    height: '',
+  },
+  base: ({width, height}) => ({
+    width,
+    height,
+    '& [data-part="graphic-img"]': {
+      maxWidth: '100%',
+      maxHeight: '100%',
+    },
+  }),
+});
```

Comment:
I'm happy to come up with a different solution. I just didn't want the `img` to extend `svgStencil` because `width` and `height` didn't do anything. I created a new stencil to break apart those styles. Does that make sense?